appflavors/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    repositories {
+        jcenter()
+        google()
+        mavenLocal()
+    }
+    dependencies {
+        classpath "com.bugsnag:bugsnag-android-gradle-plugin:${project.version}"
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion Integer.parseInt(project.ANDROID_COMPILE_SDK_VERSION)
+    buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION
+
+    defaultConfig {
+        minSdkVersion Integer.parseInt(project.ANDROID_MIN_SDK_VERSION)
+        targetSdkVersion Integer.parseInt(project.ANDROID_TARGET_SDK_VERSION)
+        versionCode Integer.parseInt(project.SAMPLE_VERSION_CODE)
+        versionName project.SAMPLE_VERSION_NAME
+    }
+
+    flavorDimensions "regular"
+
+    productFlavors {
+        foo {
+            applicationIdSuffix ".foo"
```

Comment:
These don't seem to be referenced at all in the build notification, only the proguard upload, is that right?